### PR TITLE
Create Find_Day_of_a_given_Date.md

### LINKED ISSUE #8438: Find_Day_of_a_given_Date
### Description

Description
Find_Day_of_a_given_Date

### Domain

C/CPP

### Type of Contribution

Documentation

### Code of Conduct

- [X] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.

### PR DESCRIPTION
**Pull request by varnikarathee**
<hr>

## Description 📜 Find_Day_of_a_given_Date.md

Fixes #8438

<hr>

## Type of change 📝Find_Day_of_a_given_Date.md

- [X] Documentation (Content Creation in the form of codes or tutorials)

<hr>

## Domain of Contribution 📊
- [X] C/CPP
<hr>

## Checklist ✅
- [X] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [X] I have performed a self-review of my own code or work.
- [X] I have commented my code, particularly in hard-to-understand areas.
- [X] My changes generates no new warnings.
- [X] I'm GWOC'21 contributor

<hr>

### CODE REVIEW

**Comment by varnikarathee** [comment]
@vasu-1 Can i still get my pr reviewed and merged? Please can u view once.

**Comment by vasu-1** [comment]
image
remove both the hash signs ! for labels

**Review by vasu-1 — CHANGES_REQUESTED** [review]
Add your file link in the respective folder's readme.md

File: C_CPP/Competitive Programming/Find_Day_of_a_given_Date.md [hunk 1]
```diff
@@ -0,0 +1,115 @@
+# HOW TO FIND DAY OF A GIVEN DATE   
+Generally there are two approaches to find the day of a given date    
+- Zeller’s Rule   
+- Key-Value Method  
+Here we follow the first approach i.e Zeller’s Rule.  
+###  ZELLER’S RULE:  
+Zeller’s rule comes from a Zeller’s formula :  
+F=k+ [(13*m-1)/5] +D+ [D/4] +[C/4]-2*C
```
Comment (vasu-1) [inline C_CPP/Competitive Programming/Find_Day_of_a_given_Date.md:8]: put this line in the backtick

File: C_CPP/Competitive Programming/Find_Day_of_a_given_Date.md [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+k= 9  
+m= 11  
+C= 20  
+D= 21  
+Desired changes have been made in code for finding the correct output.  
+
+ ![image](https://user-images.githubusercontent.com/77229404/143466278-57fe1b53-f6c9-45b1-bad1-652603d40e6b.png)
+
+### REFERENCES:
+-https://beginnersbook.com/2013/04/calculating-day-given-date/
```
Comment (vasu-1) [inline C_CPP/Competitive Programming/Find_Day_of_a_given_Date.md:115]: [Beginnersbook](https://beginnersbook.com/2013/04/calculating-day-given-date/)

put the link like this

**Review by vasu-1 — CHANGES_REQUESTED** [review]

**Comment by vasu-1** [comment]
> @vasu-1 Can i still get my pr reviewed and merged? Please can u view once.

yes yes

**Pull request closed without merging**